encoders/protoc-gen-firebase-encoders/protoc-gen-firebase-encoders.gradle

```diff
@@ -0,0 +1,32 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+plugins {
+    id "org.jetbrains.kotlin.jvm"
+    id "java-library"
+    id "com.github.johnrengelman.shadow" version "5.2.0"
+}
+
+jar {
+    manifest.attributes "Main-Class": "com.google.firebase.encoders.proto.codegen.MainKt"
+}
+
+dependencies {
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk8:1.3.72"
+
+    implementation "com.google.protobuf:protobuf-java:3.11.4"
```

Comment:
Full proto runtime should be OK, right?

Reply:
yep, it's fine, it's only going to be used within the plugin